api/tests/unit/domain/usecases/sessions-mass-import/validate-sessions_test.js

```diff
@@ -171,6 +188,16 @@ describe('Unit | UseCase | sessions-mass-import | validate-sessions', function (
           },
         ];
 
+        sessionsImportValidationService.getUniqueCandidates.onCall(0).returns({
```

Comment:
Quand il y en a plusieurs je propose l'usage des alias plutôt que la gymnastique mentale de 0 = premier, 1 = second

```suggestion
        sessionsImportValidationService.getUniqueCandidates..onFirstCall().returns({
```

Reply:
Je le fais aussi en général, allez je corrige